cosmos/airflow/graph.py

```diff
@@ -125,6 +126,18 @@ def create_task_metadata(
             task_id = f"{node.name}_run"
             if use_task_group is True:
                 task_id = "run"
+        if node.resource_type == DbtResourceType.SOURCE:
+            task_full_name = node.unique_id[len("source.") :]
+            task_id = f"{task_full_name}_source"
+            args["select"] = f"source:{node.unique_id[len('source.'):]}"
+            args["models"] = None
+            if use_task_group is True:
+                task_id = node.resource_type.value
+            if node.has_freshness is False:
+                return TaskMetadata(
+                    id=task_id,
+                    # arguments=args,
```

Comment:
@tatiana Returning an EmptyOperator just seemed "grayish" in the dag UI. Looking at all the cool cosmos-related operators "DbtLocal..." "Dbt...", etc and having an EmptyOperator didn't seemed 100% cool, Should we create a dummySourceOperator that inherits EmptyOperator just for the sake of having all Operators follow the same naming pattern?